user_tools/src/spark_rapids_pytools/cloud_api/databricks_azure.py

```diff
@@ -103,6 +104,14 @@ def get_supported_gpus(self) -> dict:
             gpu_scopes[mc_prof] = NodeHWInfo(sys_info=hw_info_ob, gpu_info=gpu_info_obj)
         return gpu_scopes
 
+    def generate_cluster_configuration(self, render_args: dict):
+        executor_names = ','.join([
+            f'{{"node_id": "12345678900{i}"}}'
```

Comment:
Is there a reason why using prefix `12345678900`?

Reply:
This is chosen at random.